app/workers/repository_import_worker.rb

```diff
@@ -20,6 +26,8 @@ def perform(project_id)
                             Gitlab::BitbucketImport::Importer.new(project).execute
                           elsif project.import_type == 'google_code'
                             Gitlab::GoogleCodeImport::Importer.new(project).execute
+                          elsif project.import_type == 'fogbugz'
+                            Gitlab::FogbugzImport::Importer.new(project).execute
```

Comment:
This `if/elsif` statement is getting so long that I'd prefer a `case/when` statement. Could you do that?